### PR TITLE
fix(condo): DOMA-8653 fixed table filters

### PR DESCRIPTION
**Pull request by Alllex202**
## Before

https://github.com/open-condo-software/condo/assets/56914444/498c26bf-ad37-40f2-979f-05a96ff9b576

## After

https://github.com/open-condo-software/condo/assets/56914444/1c88e5ec-026a-4f40-bb08-a0b902557392

### CODE REVIEW

**Review by sitozzz — COMMENTED** [review]
The videos does not show any positive improvements. Does really useMemo works here as a solution?

File: apps/condo/domains/common/components/Table/Index.tsx [hunk 1]
```diff
@@ -134,7 +134,7 @@ export const Table: React.FC<ITableProps> = ({
             const newParameters = getFiltersQueryData(newFilters, newSorters, newOffset)
             return updateQuery(router, { newParameters }, { resetOldParameters: false })
         }
-    }, 400)
+    }, 400), [applyQuery, filters, router, rowsPerPage, sorters])
```
[the remark this answers is not in the document]
Comment (Alllex202) [reply apps/condo/domains/common/components/Table/Index.tsx:137]: Is it really 800? 🙃
looks like this needs to be fixed
Снимок экрана 2024-03-28 в 17 56 53

[the remark this answers is not in the document]
Comment (Alllex202) [reply apps/condo/domains/common/components/Table/Index.tsx:137]: The search on the page uses the value 400, so I'll leave it as is

**Comment by Alllex202** [comment]
> The videos does not show any positive improvements. Does really useMemo works here as a solution?

The first video shows that the characters are erased during printing. There is no such behavior in the second video.

**Comment by sitozzz** [comment]
> > The videos does not show any positive improvements. Does really useMemo works here as a solution?
> 
> The first video shows that the characters are erased during printing. There is no such behavior in the second video.

Oh, now I see what happend. For that reason that might be a solution

File: apps/condo/domains/common/components/Table/Index.tsx (hunk 1 above)
[the remark this answers is not in the document]
Comment (sitozzz) [reply apps/condo/domains/common/components/Table/Index.tsx:137]: > Is it really 800? 🙃 looks like this needs to be fixed
Yup, but maybe not in this PR)

**Review by sitozzz — APPROVED** [review]

**Pull request merged**